docs/source/guides/logical_types_and_semantic_tags.ipynb

```diff
@@ -0,0 +1,629 @@
+    "\n",
+    "It is important to have the most accurate typing information for a dataset, as it impacts how the data is parsed, transformed, and interpreted downstream of Woodwork initialization. \n",
+    "\n",
+    "This guide offers an in-depth walk-through of all of the logical types and semantic tags that Woodwork defines in order to allow users to choose the logical types and semantic tags that most closely describe their data. As a reminder, here are quick definitions of Woodwork's types:\n",
+    "\n",
+    "- Physical Type: defines how the data is stored on disk or in memory.\n",
+    "- Logical Type: defines how the data should be parsed or interpreted.\n",
+    "- Semantic Tag(s): provides additional data about the meaning of the data or how it should be used.\n",
+    "\n",
+    "For an in-depth guide on how to set and manipulate these types, see the [understanding types and tags guide](https://woodwork.alteryx.com/en/stable/guides/understanding_types_and_tags.html).\n",
+    "\n",
+    "For information on how to customize Woodwork's type system, see the [custom types and inference guide](https://woodwork.alteryx.com/en/stable/guides/custom_types_and_type_inference.html).\n",
+    "\n",
+    "It's important to remember that Woodwork columns will always have a logical type and that any semantic tags that are added by Woodwork are meant to add additional meaning onto the logical type. We'll start out by looking in-depth at semantic tags so that when we get to logical types, we can better understand how a semantic tag might add additional information onto it."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "czech-strip",
+   "metadata": {},
+   "source": [
+    "\n",
+    "# Semantic Tags\n",
+    "\n",
+    "Here is the full set of Woodwork-defined semantic tags:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "supposed-cookie",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import woodwork as ww\n",
+    "ww.list_semantic_tags()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "extra-husband",
+   "metadata": {},
+   "source": [
+    "#### Standard Tags\n",
+    "Standard tags are associated with specific logical types. They are useful for indicating predefined categories that logical types might fall into.\n",
+    "- `'numeric'` - Is applied to any numeric logical type\n",
+    "    - **Uses**: Can select for just numeric columns when performing operations that require numeric operations\n",
+    "    - **Related Properties**: `series.ww.is_numeric`\n",
+    "- `'category'` - Is applied to any logical type that is categorical in nature\n",
+    "    - **Uses**: Can select for just categorical columns when performing operations that require categories\n",
+    "    - **Related Properties**: `series.ww.is_categorical`\n",
```

Comment:
These lines are not being broken up into separate lines in the rendered document